src/Pdf/DataAccess.ts

```diff
@@ -9,6 +9,17 @@ export async function getStaticMap(
 	points: StaticMapPoint[],
 ): Promise<StaticMap> {
 	const bounds: Bounds = calculateLocationsBounds(points);
+
+	if (
```

Comment:
This whole condition could be simplified to `bounds.north - bounds.south < .0002 || bounds.east - bounds.west < .0002`